usaspending_api/awards/models_matviews/mv_search_award_idv.py

```diff
@@ -85,4 +85,4 @@ class UniversalAwardView(models.Model):
 
     class Meta:
         managed = False
-        db_table = "universal_award_matview"
+        db_table = "mv_search_award_idv"
```

Comment:
Why is this just a rename from the origonal (and named "mv" instead of "my") when all the others seem to be copies?

Reply:
First I should clarify the new naming convention, "mv" is a prefix for "materialized view". I wrote notes in the PR description, which aren't enough long term but for now are as good as it gets. Eventually all of the db matviews and views will follow the new convention.

Secondly, all of the `mv_` models are identical except for the model name and db object name. It doesn't matter which model was the copy since they are identical.

Reply:
If you might use an arbitrary model, the my previous comment about having an _abstact_ base class may be better suited just as a base class (non-abstract). Then the base class can be used in the general case to carry data, and specific sub-classes can be used to do the mapping to the specific db table.